packages/http-client-js/test/scenarios/encoding/bytes_body.md

```diff
@@ -42,3 +42,48 @@ export async function foo(
   throw createRestError(response);
 }
 ```
+
+# only: Should encode a bytes data when the body is bytes and encoding is base64
+
+## TypeSpec
+
+```tsp
+@service
+namespace Test;
+
+@route("/default")
+op foo(@header contentType: "application/jsonl", @body value: bytes): {
+  @header contentType: "application/jsonl";
+  @body value: bytes;
+};
```

Comment:
@joheredi you can check out here to find out the output uses encodeUint8Array and decodeBase64 for bytes as request and response which I am not sure if it's correct or not. But if you checkout the jsonl.md in streams folder for both request and response, you will find out they didn't use encodeUint8Array in request.

Reply:
This should be due to the encoding context